lib/appsignal/plug.ex

```diff
@@ -161,14 +161,8 @@ if Appsignal.plug?() do
 
     @fallback_request_headers ~w(
       accept accept-charset accept-encoding accept-language cache-control
-      connection content-length user-agent from negotiate pragma referer range
-
-      auth-type gateway-interface path-translated remote-host remote-ident
-      remote-user remote-addr request-method server-name server-port
-      server-protocol request-uri path-info client-ip range
-
-      x-request-start x-queue-start x-queue-time x-heroku-queue-wait-time
-      x-application-start x-forwarded-for x-real-ip
+      connection content-length path-info range request-method request-uri
+      server-name server-port server-protocol
     )
```

Comment:
Wouldn't it be better to move this to the default config?
So we keep the config in one place, for when we support other web frameworks in the future?